src/data/FAQs.json

```diff
@@ -0,0 +1,56 @@
+{
+  "qa": [
+    {
+      "id": 1,
+      "section": "<h1>Recruitment</h1>",
```

Comment:
Nice attempt! Since our section headers and questions are kinda fixed in terms of styling, let's just use string for them, eg.` "section": "Recruitment"` and `"Q": "Who do we reach out to if we have more enquiries?"`. So we don't need to do any html to string conversion for sections and questions, **but only for answers**

Reply:
Also noticed you added an id in `<p id='main'>` of each answer, not sure if that is necessary. If it isn't, can remove too.